DeUrgenta/Features/EmergencyBackpack/ViewModel/CategoryViewModel.swift

```diff
@@ -0,0 +1,76 @@
+import Foundation
+import SwiftUI
+
+class CategoryViewModel: ObservableObject {
+    @Published var showingAlert = false
+    @Published var name: String = ""
+    @Published var quantity: Int = 1
+    @Published var expirationDate: Date = .init()
+
+    @Published var selectedCategory: CategoryItem?
+    @Published var currentItemsArray: [Item] = []
+
+    @Published var hasItem: Bool = false
+    @Published var hideSection: Bool = false
+
+    init(selectedCategory: CategoryItem? = nil) {
+        self.selectedCategory = selectedCategory
+    }
+
+    let items = [
+        CategoryItem(title: AppStrings.CategoryItem.foodTitle.localized(), icon: "icon-syrup", type: .food),
+        CategoryItem(title: AppStrings.CategoryItem.hygieneItemsTitle.localized(), icon: "icon-cotton-swabs", type: .hygieneItems),
+        CategoryItem(title: AppStrings.CategoryItem.firstAidKitTitle.localized(), icon: "icon-capsules", type: .firstAidKit),
+        CategoryItem(title: AppStrings.CategoryItem.docsTitle.localized(), icon: "icon-prescription", type: .docs),
+        CategoryItem(title: AppStrings.CategoryItem.survivalItemsTitle.localized(), icon: "icon-medical-mask", type: .survivalItems),
+        CategoryItem(title: AppStrings.CategoryItem.mixedTitle.localized(), icon: "icon-pharmacy", type: .mixed),
+    ]
+
+    func addItem() {
+        if let selectedCategory = selectedCategory {
```

Comment:
Here I think is more appropriate to use a guard statement. With guard we can exit earlier and the code looks better